cli/command/image/push_test.go

```diff
@@ -69,3 +69,26 @@ func TestNewPushCommandSuccess(t *testing.T) {
 		assert.NilError(t, cmd.Execute())
 	}
 }
+
+func TestNewQuietPushCommandSuccess(t *testing.T) {
+	testCases := []struct {
+		name string
+		args []string
+	}{
+		{
+			name: "simple",
+			args: []string{"--quiet", "image:tag"},
+		},
+	}
+	for _, tc := range testCases {
```

Comment:
Thank you for this test! But I think you can safely remove this `for pattern` as you have only one test case 😸

Reply:
Thank you! Will change pr within the hour.